frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -54,8 +54,30 @@ const props = defineProps({
         type: Array as PropType<FamAccessControlPrivilegeGetResponse[]>,
         default: [],
     },
+    newUserAccessIds: {
+        type: String,
+        default: '',
+    },
+    newAppAdminId: {
+        type: String,
+        default: '',
+    },
+    newDelegatedAdminIds: {
```

Comment:
Hi Jota, these names look good, but we don't need three props, we can just use one name for all three cases. They are doing the same thing, and we should always keep the code simple as we can.  Let's just call it `newUserAccessIds`, default = [], so for app admin, pass [app_admin_id]

Reply:
as discussed, we need to tell the new ids are for which table, so it's okay to keep three for now, thanks!!